include/SDL3/SDL_gpu.h

```diff
@@ -2257,6 +2260,123 @@ extern SDL_DECLSPEC const char * SDLCALL SDL_GetGPUDeviceDriver(SDL_GPUDevice *d
+ *   Contains the self-reported version of the underlying system driver. This is
+ *   a relatively short version string in an unspecified format. If
+ *   SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_INFO_STRING is available then that
+ *   property should be preferred over this one as it may contain additional
+ *   information that is useful for identifying the exact driver version used.
+ *
+ *   Strings that have been found in the wild include:
+ *     53.0.0
+ *     0.2.2019
+ *     0.405.2463
+ *     572.91.0.0
+ *     32.0.15.6614
+ *
+ * SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_INFO_STRING:
+ *   Contains the detailed version information of the underlying system driver
+ *   as reported by the driver. This is an arbitrary string with no standardized
+ *   format and it may contain newlines. This property should be preferred over
+ *   SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_VERSION_STRING if it is available as it
+ *   usually contains the same information but in a format that is easier to
+ *   read.
+ *
+ *   Strings that have been found in the wild include:
+ *     566.14
+ *     101.6559
+ *     1.2.11
+ *     Mesa 21.2.2 (LLVM 12.0.1)
+ *     Mesa 22.2.0-devel (git-f226222 2022-04-14 impish-oibaf-ppa)
+ *     Mesa 25.1.0-devel (git-7e2c3be454)
+ *     v1.r53p0-00eac0.824c4f31403fb1fbf8ee1042422c2129
+ *
+ *   As well as the multiline string (which has a trailing newline):
+ *     Driver Build: 85da404, I46ff5fc46f, 1606794520
+ *     Date: 11/30/20
+ *     Compiler Version: EV031.31.04.01
+ *     Driver Branch: promo490_3_Google
+ *
+ * SDL_PROP_GPU_DEVICE_DEBUG_VULKAN_CONFORMANCE_STRING:
+ *   When using the Vulkan backend, contains the highest Vulkan version number
+ *   that the system driver claims that the underlying hardware conforms to.
+ *   This is self-reported and may not be truthful.
+ *
+ *   Strings that have been found in the wild include:
+ *     0.0.0.0
+ *     1.0.0.0
+ *     1.1.2.0
+ *     1.3.8.2
+ *     1.4.0.0
+ * ```
+ *
+ * \param device a GPU context to query.
+ * \returns a valid property ID or 0. A value of 0 indicates that no properties
+ *          are available and is not an error. The returned property group is
+ *          owned by SDL and has the same lifetime as the containing context. It
+ *          should not be destroyed manually when no longer needed.
+ *
+ * \since This function is available since SDL 3.4.0.
+ */
+extern SDL_DECLSPEC SDL_PropertiesID SDLCALL SDL_GetGPUDeviceDebugProperties(
+    SDL_GPUDevice *device);
+
+#define SDL_PROP_GPU_DEVICE_DEBUG_NAME_STRING               "SDL.gpu.device.debug.name"
+#define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_NAME_STRING        "SDL.gpu.device.debug.driver_name"
+#define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_VERSION_STRING     "SDL.gpu.device.debug.driver_version"
+#define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_INFO_STRING        "SDL.gpu.device.debug.driver_info"
+#define SDL_PROP_GPU_DEVICE_DEBUG_VULKAN_CONFORMANCE_STRING "SDL.gpu.device.debug.vulkan_conformance"
```

Comment:
I wonder if it makes sense to namespace-ify the backend name, i.e.

```suggestion
#define SDL_PROP_GPU_DEVICE_DEBUG_VULKAN_CONFORMANCE_STRING "SDL.gpu.device.debug.vulkan.conformance"
```

This may matter later for things like D3D shader model version.